website/docs/r/dx_connection.html.markdown

```diff
@@ -20,12 +22,41 @@ resource "aws_dx_connection" "hoge" {
 }
 ```
 
+### Request a MACsec-capable connection
+
+```terraform
+resource "aws_dx_connection" "example" {
+  name             = "tf-dx-connection"
+  bandwidth        = "10Gbps"
+  location         = "EqDA2"
+  macsec_requested = true
```

Comment:
The API uses a parameter name of `requestMACSec`. Consider using `request_macsec` as the field name for the schema as well.
```suggestion
  request_macsec = true
```